storage/cuprate-blockchain/src/service/read.rs

```diff
@@ -470,11 +504,6 @@ fn check_k_is_not_spent(env: &ConcreteEnv, key_images: HashSet<KeyImage>) -> Res
         key_image_exists(&key_image, tables.key_images())
     };
 
-    // FIXME:
-    // Create/use `enum cuprate_types::Exist { Does, DoesNot }`
-    // or similar instead of `bool` for clarity.
-    // <https://github.com/Cuprate/cuprate/pull/113#discussion_r1581536526>
-    //
```

Comment:
Btw I think this code is conflicting with the surrounding comments and docs here: https://github.com/Cuprate/cuprate/blob/unify-db-enums/types/src/service.rs#L161-L165 (my fault).

`enum Exist { Does, DoesNot }` or similar would solve this.

Reply:
added this todo back